### PR TITLE
feature/6933-Dylan-e2eiOSSlackNotifications

### PR DESCRIPTION
**Pull request by Sparowhawk**
## Description of Change
turned on slack thread and daily qa schedule for iOS e2e test

## Testing
yarn test

- [x] Tested on iOS 
- [x] Tested on Android 

## Reviewer Validations
turned on slack thread and daily qa schedule for iOS e2e test

## PR Checklist

  **Reviewer:** Confirm the items below as you review

- [x] PR is connected to issue(s)
- [x] Tests are included to cover this change (when possible)
- [x] No magic strings (All string unions follow the [Union -> Constant](https://github.com/department-of-veterans-affairs/va-mobile-app/blob/develop/VAMobile/src/constants/common.ts) type pattern)
- [x] No secrets or API keys are checked in
- [x] All imports are absolute (no relative imports)
- [x] New functions and Redux work have proper TSDoc annotations

## For QA

[Run a build for this branch](https://github.com/department-of-veterans-affairs/va-mobile-app/actions/workflows/on_demand_build.yml)

### CODE REVIEW

**Review by theodur — CHANGES_REQUESTED** [review]
Same comments from https://github.com/department-of-veterans-affairs/va-mobile-app/pull/7199#pullrequestreview-1741034320 apply here

**Comment by Sparowhawk** [comment]
> Same comments from [#7199 (review)](https://github.com/department-of-veterans-affairs/va-mobile-app/pull/7199#pullrequestreview-1741034320) applies here

updated and will combine with the CU ticket for the singular slack thread

File: .github/workflows/e2e_ios.yml [hunk 1]
```diff
@@ -22,9 +27,21 @@ env:
...
 
 jobs:
+  start_slack_thread:
+    if: github.event_name == 'schedule'
+    name: Start Slack thread
+    uses: ./.github/workflows/start_slack_thread.yml
+    secrets: inherit
+    with:
+      channel_name: va-mobile-build-alerts
+      message: 'E2E iOS build process starting. Please see :thread: for results. This process may take a while.'
```
Comment (theodur) [inline .github/workflows/e2e_ios.yml:41]: Had a similar suggestion for the messaging here too
https://github.com/department-of-veterans-affairs/va-mobile-app/pull/7199#discussion_r1408135342

Comment (Sparowhawk) [reply .github/workflows/e2e_ios.yml]: updated

**Review by theodur — CHANGES_REQUESTED** [review]
Noticed a typo

File: .github/workflows/e2e_ios.yml [hunk 2]
```diff
@@ -22,9 +27,21 @@ env:
   APP_CLIENT_SECRET: ${{ secrets.APP_CLIENT_SECRET }}
   # IAM production app client secret
   APP_CLIENT_SECRET_PROD: ${{ secrets.APP_CLIENT_SECRET_PROD }}
+  # Slack API token used by fastlane
```
[the remark this answers is not in the document]
Comment (Sparowhawk) [reply .github/workflows/e2e_ios.yml]: updated

**Review by theodur — APPROVED** [review]

**Review by dumathane — APPROVED** [review]

**Pull request merged**